test_cuda_detailed.py

```
@@ -0,0 +1,136 @@
+        # Check if we can find CUDA libraries
+        print("   Searching for CUDA libraries in LD_LIBRARY_PATH...")
+        ld_path = os.environ.get('LD_LIBRARY_PATH', '')
+        cuda_libs_found = []
+        for path in ld_path.split(':'):
+            if os.path.exists(path):
+                try:
+                    files = os.listdir(path)
+                    cuda_files = [f for f in files if 'cuda' in f.lower()]
+                    if cuda_files:
+                        cuda_libs_found.extend([os.path.join(path, f) for f in cuda_files[:3]])  # Show first 3
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)